bot/seasons/christmas/hanukkah_embed

```diff
@@ -0,0 +1,110 @@
+from discord.ext import commands
+from discord import Embed
+import requests
+import datetime
+import logging
+
+
+log = logging.getLogger(__name__)
+
+HANUKKAH_DAYS = []
+HANUKKAH_MONTHS = []
+HANUKKAH_YEARS = []
+
+
+class HanukkahEmbed(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.url = """https://www.hebcal.com/hebcal/?v=1&cfg=json&maj=on&min=on&mod=on&nx=on&year=now&month=x&ss=on&
+                      mf=on&c=on&geo=geoname&geonameid=3448439&m=50&s=on"""
+        self.hanukkah_dates = self.get_hanukkah_dates()
+
+    def get_hanukkah_dates(self):
```

Comment:
doc string for every methods and classes

Reply:
on it